services/projects-service/serverless.functions.yml

```
@@ -115,3 +115,13 @@ getImagingProjectStatus:
           cors: true
           authorizer:
             arn: "#{COGNITO_USER_POOL_ARN}"
+reimport:
+    handler: src/reimportImagingProjectStudies.handler
+    role: "#{LAMBDA_ROLE}"
+    environment:
+        ENVIRONMENT: "#{ENVIRONMENT}"
+    events:
+        - schedule:
+              # Set agreed interval
+              rate: rate(60 minutes)
```

```suggestion
              rate: #{CHECK_REIMPORT_RATE}
```
It won't be enough to just call this lambda every so often. We'll need to add in another variable that defines the time after which a project's studies should be reimported